src/jdk.jartool/share/classes/sun/tools/jar/Main.java

```diff
@@ -1761,14 +1782,28 @@ private File createTemporaryFile(String tmpbase, String suffix) {
     static class StreamedModuleInfoEntry implements ModuleInfoEntry {
         private final String name;
         private final byte[] bytes;
-        StreamedModuleInfoEntry(String name, byte[] bytes) {
+        private final Long lastModifiedTime;
```

Comment:
It might be better to use a FileTime here, otherwise we risk a NPE when unboxing.

Reply:
Sounds good. I've updated the PR to replace this to use `FileTime`.